todo/tests/fixtures/google_oauth.py

```
@@ -0,0 +1,6 @@
+GOOGLE_OAUTH_FIXTURE = {
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)